engine/schema/src/main/java/org/apache/cloudstack/backup/BackupVO.java

```diff
@@ -82,6 +87,9 @@ public class BackupVO implements Backup {
     @Column(name = "zone_id")
     private long zoneId;
 
+    @Column(name = "backed_volumes", length = 65535)
+    protected String backedVolumes;
```

Comment:
What does this store? UUID or path? Would it have been better to use a details or backup_volume_map table?

Reply:
I've followed the convention of the backup framework. A similar column is present in the vm_instance table.